Extensions/xkit_preferences.js

```diff
@@ -1810,7 +1815,15 @@ XKit.extensions.xkit_preferences = new Object({
 					m_extra_classes = "xkit-experimental-option";
 				}
 
+				var m_extra_style = "";
+				if (XKit.extensions[extension_id].preferences[pref].mobile_only === true && XKit.browser().mobile === false) {
+					m_extra_style = "display: none;";
+				} else if (XKit.extensions[extension_id].preferences[pref].desktop_only === true && XKit.browser().mobile === true) {
```

Comment:
I would strongly disagree with setting up xkit_developers this way.
everything in-tree should ideally be identified as a new-xkit extension.
there's no point in putting individual author names on this project.

(I'm guessing this was an accident on @dlmarquis's part?)

On Wed, Aug 19, 2015 at 9:48 PM James Hobin <notifications@github.com>
wrote:

> In Extensions/xkit_preferences.js
> <https://github.com/new-xkit/XKit/pull/322#discussion_r37490821>:
>
> > @@ -1810,7 +1815,15 @@ XKit.extensions.xkit_preferences = new Object({
> >  					m_extra_classes = "xkit-experimental-option";
> >  				}
> >
> > +				var m_extra_style = "";
> > +				if (XKit.extensions[extension_id]
>
> .preferences[pref].mobile_only === true && XKit.browser().mobile === false) {
> > +					m_extra_style = "display: none;";
> > +				} else if (XKit.extensions[extension_id].preferences[pref].desktop_only === true && XKit.browser().mobile === true) {
>
> Same as above
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/322/files#r37490821>.
>